### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/get-started/env.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#696] [Announcements]
# https://theme-hope.vuejs.press/zh/get-started/env.html

本教程将指导你如何搭建 VuePress 的运行环境。

https://theme-hope.vuejs.press/zh/get-started/env.html

**Comment by sunheyi6** [comment]
下载链接 404了

**Reply by esyka114514** [reply]
other downlaod

**Comment by shilic** [comment]
执行 `corepack enable` 命令需要在管理员用户下，如果是windows环境，请右键以管理员身份运行命令提示符
